libsplinter/src/node_id/store/diesel/mod.rs

```diff
@@ -28,7 +28,7 @@ use operations::{
     get_node_id::NodeIdGetOperation, set_node_id::NodeIdSetOperation, NodeIdOperations,
 };
 
-/// Database backed NodeIdStore implementation.
+/// Database backed [NodeIdStore](../trait.NodeIdStore.html) implementation.
```

Comment:
Since this trait is imported, you should be able to just wrap the type name in a square brackets and the doc generator will figure it out.

Reply:
fixed